api/eslint.config.js

```diff
@@ -1,13 +1,20 @@
 import globals from 'globals';
 import pluginJs from '@eslint/js';
+import pluginTs from 'typescript-eslint';
+// import pluginUnicorn from 'eslint-plugin-unicorn';
 
-/** @type {import('eslint').Linter.Config[]} */
-export default [
+// https://typescript-eslint.io/getting-started
+export default pluginTs.config(
+  pluginJs.configs.recommended,
+  pluginTs.configs.recommended,
+  // pluginUnicorn.configs['flat/recommended'], // TODO activate unicorn
   {
-    languageOptions: { globals: globals.node },
+    files: ['src/**/*.ts', 'src/**/*.js'],
```

Comment:
we don't have a .js file in api/ folder, maybe in cli/tempaltes
```suggestion
    files: ['src/**/*.ts'],
```

Reply:
correct, we used to thank you